planetarium/NineChronicles
# Add option tag on ItemView.

### Description

SSIA

### How to test

1. Make an elemental equipment.
2. If it has additional stats or skills, check if `ItemView` shows their count.

### Required Reviewers

@planetarium/9c-dev 

### Screenshot
![image](https://user-images.githubusercontent.com/26648527/124891170-b2945c80-e013-11eb-8985-450ea88c0eba.png)
